records_mover/db/redshift/unloader.py

```diff
@@ -92,7 +93,6 @@ def unload(self,
                table: str,
                unload_plan: RecordsUnloadPlan,
                directory: RecordsDirectory) -> int:
-        logger.info("Starting Redshift unload...")
```

Comment:
I wanted to make this one more specific on exactly where the data is going, as the unloads can take some time for larger tables and being able to watch the location can be helpful for folks.